src/pages/Route/transform.ts

```diff
@@ -53,11 +53,17 @@ export const transformStepData = ({
     protocols = protocols.concat('websocket');
   }
 
+  let { status } = step1Data;
+  if (step1Data.status === undefined) {
```

Comment:
status should be always a number or string flag I think, why this will have the `undefined` case? 🤔

Reply:
done